rustup/rust.go

```diff
@@ -79,6 +80,13 @@ func (r Rust) Contribute(layer libcnb.Layer) (libcnb.Layer, error) {
 	layer, err := r.LayerContributor.Contribute(layer, func() (libcnb.Layer, error) {
 		r.Logger.Body("Installing Rust")
 
+		// This layer doesn't actually contain files, they write to RUSTUP_HOME, because Rustup is installing them.
+		// Because this layer build + cache and it is empty, libpak & the LayerContributor will think it's a problem and reload it.
+		// To bypass that, we are writing an empty file to the layer to prevent that from happening.
+		if err := ioutil.WriteFile(filepath.Join(layer.Path, "marker"), []byte{}, 0644); err != nil {
```

Comment:
```suggestion
		if err := os.WriteFile(filepath.Join(layer.Path, "marker"), []byte{}, 0644); err != nil {
```
`ioutil` is deprecated.

Reply:
Old habits... 👍